.evergreen/releases.yml

```diff
@@ -40,45 +74,82 @@ functions:
           rm expansion.yml
 
   "install dependencies":
-    command: shell.exec
-    params:
-      working_dir: "src"
-      script: |
-        ${PREPARE_SHELL}
-        .evergreen/install-dependencies.sh rust
+    - command: shell.exec
+      params:
+        working_dir: "src"
+        script: |
+          ${PREPARE_SHELL}
+          .evergreen/install-dependencies.sh rust
+
+    - command: subprocess.exec
+      params:
+        working_dir: src
+        include_expansions_in_env:
+          - DRIVERS_TOOLS
+        binary: bash
+        args:
+          - .evergreen/fetch-drivers-tools.sh
+
+  "build papertrail vars":
+    - command: subprocess.exec
+      params:
+        working_dir: src
+        include_expansions_in_env:
```

Comment:
In general I think it's better to use `include_expansions_in_env` rather than `add_expansions_to_env: true` - it makes the data dependencies explicit and is handy when grepping for a variable.